app/models/api_key.rb

```diff
@@ -16,6 +17,24 @@ class KeyAccessError < StandardError; end
 
   after_initialize :generate_key
 
+  def self.scope_mappings
+    @mappings ||= {
+      topics: {
+        write: { action: 'posts#create', params: %i[topic_id] },
```

Comment:
Can we make it possible for `action` to be an array? So that later we have the option of making topics:write support multiple actions?